src/erlcloud_inspector.erl

```diff
@@ -134,16 +134,16 @@ inspector_result_fun(#aws_request{response_type = error, error_type = aws} = Req
 -type attribute() :: proplists:proplist().
 
 -spec add_attributes_to_findings
-          (Attributes :: [attribute()],
-           FindingArns :: [string()]) ->
+          (Attributes :: attribute(),
+           FindingArns :: [string() | binary()]) ->
```

Comment:
All of these `| binary()` respect the tests (I didn't want to break interface) but don't know what is preferable to keep or let go.

Reply:
What you have looks correct, all the stuff calls `to_binary/1` which allows strings or binary strings. Just incomplete spec. Thanks.